Library/Homebrew/formulary.rb

```diff
@@ -331,6 +331,15 @@ def self.core_path(name)
     CoreTap.instance.formula_dir/"#{name.downcase}.rb"
   end
 
+  # Unlike tap_paths, this method returns the path to the named formula if it
+  # is to be created within the given tap. Whether the formula is already in
+  # the tap is not checked.
+  def self.tap_path(name, tap)
+    user, repo = tap.split("/", 2)
+    tap_instance = Tap.new(user, repo)
```

Comment:
Probably doesn't matter much, as my other code comments suggest to eliminate this code. But you don't normally want to use `Tap.new`. Always use `Tap.fetch`, which will make sure to return the correct object even if `homebrew/core` is specified. It will also automatically split the string for you, if you pass a single argument.